docs/operation_center/events.md

```diff
@@ -5,3 +5,38 @@ The events page diplays a list of latest events received by SEKOIA.IO. You can s
 On the event page, you can access the raw format of your events, as received by SEKOIA.IO and the STIX format in which it is transformed.
 
 ![Event Page](../assets/operation_center/events.gif)
+
+## Log list:
+The scrolling list displays all the logs you have sent us.
+The default columns:
+- Timestamp: Event date
+- Event.dialect: Type of intake that sent the event.
+- Description: Smart description with clickable links formatted by SEKOIA.IO to show the most important elements and make them easily accessible such as (IP Address, Type, Name)
+
+
+## Log lines:
+Each line of log can be unrolled to show:
+- Details: Detailed information retrieved from the log about all events once they are parsed and elements related to the intake and community, using the event page searchable format.
+- STIX: Event as a STIX bundle that will be used by detection engines.
+- Raw event: Event as received.
+
+## Search bar:
+
+### Filters:
+It is possible to use and combine filters in the search bar by using:
+- Smart description: By clicking on the "+" button.
+- Details: The query is made on the existing fields in the "details" section by clicking on the "+" button for one of the items.
+To go back to the list of logs, you need to clear filters and select "Current events"
```

Comment:
I think we should review the beginning of this help file. Paragraph concerning the dork language should be inserted here, and the first screen should be updated, it doesn't correspond to the actual display.